src/cartesi-machine.lua

```diff
@@ -292,14 +292,137 @@ where options are:
   --max-uarch-cycle=<number>
     stop at a given micro cycle.
 
+  --unreproducible
```

Comment:
This does not automatically enable htif console getchar nor virtio console, right?

Reply:
Correct, this option by itself will not trigger any console or terminal related polling.